src/cell/conventional_cell.js

```diff
@@ -0,0 +1,85 @@
+import math from "../math";
+import {LATTICE_TYPE} from "../lattice/types";
+
+/**
+ * Routines for calculating conventional cell vectors from primitive cell Bravais parameters.
+ * Following Setyawan, W., & Curtarolo, S. (2010). doi:10.1016/j.commatsci.2010.05.010
+ */
+
+const unitMatrix = [
+    [1, 0, 0],
+    [0, 1, 0],
+    [0, 0, 1]
+];
+
+const CONVENTIONAL_TO_PRIMITIVE_CELL_MULTIPLIERS = {
+
+    [LATTICE_TYPE.CUB]: unitMatrix,
+
+    [LATTICE_TYPE.FCC]: [
+        [-1, 1, 1],
+        [1, -1, 1],
+        [1, 1, -1]
+    ],
+
+    [LATTICE_TYPE.BCC]: [
+        [0, 1, 1],
+        [1, 0, 1],
+        [1, 1, 0]
+    ],
+
+    [LATTICE_TYPE.TET]: unitMatrix,
+
+    [LATTICE_TYPE.BCT]: [
+        [0, 1, 1],
+        [1, 0, 1],
+        [1, 1, 0]
+    ],
+
+    [LATTICE_TYPE.ORC]: unitMatrix,
+
+    [LATTICE_TYPE.ORCF]: [
+        [-1, 1, 1],
+        [1, -1, 1],
+        [1, 1, -1]
+    ],
+
+    [LATTICE_TYPE.ORCI]: [
+        [0, 1, 1],
+        [1, 0, 1],
+        [1, 1, 0]
+    ],
+
+    [LATTICE_TYPE.ORCC]: [
+        [1, -1, 0],
+        [1, 1, 0],
+        [0, 0, 1]
+    ],
+
+    [LATTICE_TYPE.HEX]: unitMatrix,
+
+    [LATTICE_TYPE.RHL]: unitMatrix,
+
+    [LATTICE_TYPE.MCL]: unitMatrix,
+
+    [LATTICE_TYPE.MCLC]: [
+        [1, -1, 0],
+        [1, 1, 0],
+        [0, 0, 1]
```

Comment:
Are you sure about this one and ORCC above? Just doublechecking

Reply:
It seems as they have the same representations.